src/Platform/Microsoft.Testing.Platform/OutputDevice/TerminalOutputDevice.cs

```diff
@@ -115,7 +115,13 @@ public TerminalOutputDevice(ITestApplicationCancellationTokenSource testApplicat
             // The test host controller info is not setup and populated until after this constructor, because it writes banner and then after it figures out if
             // the runner is a testHost controller, so we would always have it as null if we capture it directly. Instead we need to check it via
             // func.
-            : () => testHostControllerInfo.IsCurrentProcessTestHostController == null ? null : !testHostControllerInfo.IsCurrentProcessTestHostController;
+            : () => _isServerMode
+                ? false
+                : _isListTests
```

Comment:
Approving as-is, and maybe it's my F# experience talking here but a switch pattern would probably be easier to read here.